_submit/submit-your-performance.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: abouts
+layout: submit
```

Comment:
FWIW: all these sub pages don't really need their own layout file and could just use the `page` layout...

Reply:
also, these sub pages with collection lists are getting underlined links where generally throughout the site collections don't have underlined links...

Reply:
did this to make the nav automatically pop down for each sub header... there might be a better way but trying to keep things using the same "form"

Reply:
ahh, gotcha. don't really need to check the layout. could do it like the others and check ` if page.permalink contains '/about/'` since all the about pages start with `/about/`